src/Components/Genre/Genre.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import classes from "./Genre.module.css"
+
+class Genre extends React.PureComponent {
+  render() {
+    const genres = this.props.genres.map((item, index) => <li key={index}>{item}</li>)
```

Comment:
Usually we try to avoid moving chunks of jsx formation into variables and prefer the following:
```
const {genres} = this.props;
      
    return (
      <div className={classes.genre}>
        <ul>
            {genres.map(item => <li key={item.id}>{item}</li>)}
        </ul>
      </div>
    )
```